Doc/library/uuid.rst

```diff
@@ -288,6 +288,24 @@ of the :attr:`~UUID.variant` attribute:
 
    Reserved for future definition.
 
+The :mod:`uuid` module defines the special Nil and Max UUID values:
```

Comment:
```suggestion

The :mod:`uuid` module defines the special Nil and Max UUID values:
```

I think we should have two blank lines although the "The :mod:\`uuid\` module defines the following constants for the possible values" part above does not satisfy this (I don't think we'll ever be able to modify that one but let's still keep the diff as small as possible)